serverless/serverless
# Explicitly Added LogGroups as CF Resources

## What did you implement:
Pulled @erikerikson branch 😊 

Closes #2126 and #1944



## How did you implement it:



## How can we verify it:



Run `serverless deploy --noDeploy` Using this config:

```yml
service: serverless-logging-core-resource

provider:
  name: aws
  cfLogs: true

functions:
  hello:
    handler: handler.hello
```
And check the "update" CF template. You'll see that there's a new LogGroup resource for each function. You can also do the following for actual testing:

- deploy
- invoke
- log
- remove => make sure the resource group was removed from AWS console

## Todos:



***Is this ready for review?:*** YES
